packages/elements/src/datetime-picker/constants.ts

```diff
@@ -0,0 +1,6 @@
+enum DateTimePickerDataName {
```

Comment:
Let's name it meaningfully
```suggestion
enum DateTimePickerTranslationKeys {
```